src/components/pageComponents/ExpandableListItem.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  ListItem,
+  ListItemText,
+  Collapse,
+  makeStyles,
+} from "@material-ui/core/";
+import { ExpandLess, ExpandMore } from "@material-ui/icons/";
+import { Link } from "react-router-dom";
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    marginStart: "16px",
+    marginEnd: "16px",
+  },
+
+  topborder: {
+    borderTop: "1.5px solid #A7525E",
+    paddingLeft: "0",
+  },
+
+  link: {
+    textDecoration: "none",
+    color: "black",
+  },
+
+  expandIcon: {
+    marginLeft: "auto",
+  },
+
+  text: {
+    textAlign: "start",
+    paddingStart: "16px",
+    marginBottom: "8px",
+    paddingEnd: "16px",
+    color: "#2E303CAA",
+  },
+}));
+
+export default function ExpandableListItem(props) {
+  const { name, description, link } = props;
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleExpandClick = () => {
+    setOpen(!open);
+  };
+
+  return (
+    <div className={classes.container}>
+      <ListItem className={classes.topborder}>
+        <Link to={link} className={classes.link}>
+          <ListItemText>{name}</ListItemText>
+        </Link>
+        {open ? (
```

Comment:
Generally I am a fan of ternary operators - but in this case I would prefer a normal if.
Imo i would only user ternary operatos on small statements which can be expressed in one line. But would also like to hear the opinion of @cetceeve on this.